.dev/clima_formatter_default_image.jl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env julia
+#
+# Called with no arguments will replace the default system image with one that
+# includes the JuliaFormatter
+#
+# Called with a single argument the system image for the formatter will be
+# placed in the path specified by the argument (relative to the callers path)
+
+using Pkg
+Pkg.add("PackageCompiler")
+using PackageCompiler
+
+if isfile(PackageCompiler.backup_default_sysimg_path())
+    @error """
+    A custom default system image already exists.
+    Either restore default with:
+        julia -e "using PackageCompiler; PackageCompiler.restore_default_sysimage()"
+    or use the script
+        $(abspath(joinpath(@__DIR__, "..", ".dev", "clima_formatter_image.jl")))
+    which will use a custom path for the system image
+    """
+    exit(1)
+end
+
+# If a current Manifest exist for the formatter we remove it so that we have the
+# latest version
+rm(joinpath(@__DIR__, "Manifest.toml"), force = true)
```

Comment:
```suggestion
rm(joinpath(@__DIR__, "Manifest.toml"); force = true)
```
It works, but I think it's going to be deprecated soon.

Reply:
Thanks